package.json

```diff
@@ -66,6 +66,7 @@
     "perf": "mocha --compilers js:babel-core/register tests/performance/s3standard.js",
     "start": "node init.js && node index.js",
     "start_utapi": "node utapiServer.js",
+    "aws_listing_test_check": "REAL_AWS=1 mocha tests/functional/listing/test.js",
```

Comment:
Let's use the setup mechanism we already have for the aws functional tests.  We can already run against aws by setting an env variable AWS_ON_AIR=true